### ISSUE TITLE
Buttons must have discernible text

### ISSUE THREAD

**Issue by ENT8R** [perliedman/leaflet-control-geocoder#262]
Users relying on screen readers might not be able to know what kind of button the this is... Maybe the placeholder of the input field should be used as `aria-label=` of the button?
See https://dequeuniversity.com/rules/axe/3.1/button-name for some more information
![aria](https://user-images.githubusercontent.com/25306497/66224569-7b727800-e6d6-11e9-99db-3ff7e0d0ddd4.png)

**Comment by perliedman** [comment]
It sounds like you might know what to actually do here, while I certainly don't. I'm all for improving accessibility of the plugin, but I simply do not have time to invest in this now. Happy to look at a PR if submitted!
